libs/partners/mongodb/langchain_mongodb/chat_message_histories.py

```diff
@@ -68,6 +68,7 @@ def __init__(
         session_id_key: str = DEFAULT_SESSION_ID_KEY,
         history_key: str = DEFAULT_HISTORY_KEY,
         create_index: bool = True,
+        doc_limit: int = None,
```

Comment:
Could you rename the parameter and fix the type signature? (it's an Optional[Int])


`doc_limit` is not a meaningful name in this context. You can do something like `max_messages` and document that this is a limit on the max number of messages that can be fetched.

---

How are you using this capability with chat history? Specifically, how are you dealing with the system message (the first message in a chat history conversation).

Reply:
Hi @eyurtsev Thanks for quick response. Your suggestions are helpful. I have changed the file according to your suggestions.

I haven't handled such scenario of dealing with initial messages. Basically I am building this code as I get in trouble while developing application. I am sure I will have to handle such cases in the future and will contribute to it soon.